### PR TITLE
BFS,DFS and Topological Sort in Python

### PR DESCRIPTION
**Pull request by AshTiwari**
Added Breadth First Search Algorithm.
Added Depth First Search Algorithm.
Added Topological Sort Algorithm.

### CODE REVIEW

File: Python/dfs.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+for v in Vertex:
+    if v not in parent.keys():
+        parent[v] = None
+        stack.append(v)
+        dfs(v)
+
+
+print('DFS order is: ')
+print(stack)
+
```
Comment (HarshCasper) [inline Python/dfs.py]: Please Remove unnecessary lines in the Code and Format it properly.

File: Python/bfs.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+#showVertex()
+#showEdges()
+
+print('Implementing BFS.')
+bfs()
+
+print(level)
+
+
+
```
Comment (HarshCasper) [inline Python/bfs.py]: Here as well. Please remove unnecessary spaces as this will worsen the Readability here.

File: Python/topological_sort.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (HarshCasper) [inline Python/topological_sort.py]: Here as well

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please resolve these Issues. And add a Comment on the respective Issues that you have resolved so that I can mark that you have completed all those.

**Comment by mar-galho** [comment]
Great! Thanks!

@all-contributors please add @AshTiwari for code.

**Pull request merged**